source/stylesheets/refills/_footer-2.scss

```diff
@@ -111,17 +93,18 @@
     }
 
     ul.footer-social {
-      margin-top: 1em;
+      margin: $small-spacing 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `$small-spacing 0 0` instead of `$small-spacing 0 0 0`